cmd/destroy/local.go

```diff
@@ -41,6 +41,10 @@ func newLocalDestroyer(
 
 func (ld *localDestroyCommand) destroy(ctx context.Context, opts *Options) error {
 
+	if ld.manifest.Destroy.Remote == true {
```

Comment:
Same here, we need to check that `ld.manifest` and `ld.manifest.Destroy` are not nil